templates/CRM/common/TabHeader.tpl

```diff
@@ -29,7 +29,12 @@
       {foreach from=$tabHeader key=tabName item=tabValue}
         {if $tabValue.template}
```

Comment:
@colemanw I just updated for consistency with here. I added `empty` to the bit below to avoid an ENOTICE, so maybe it is good to use for both? Or is `isset` preferable?

Reply:
IIRC empty() is not compatible with smarty escaping so it's better to use isset(). I think? Might wanna double check what other templates are doing these days.

Reply:
https://docs.civicrm.org/dev/en/latest/security/outputs/#gotcha-2-smarty-notices

isset = BAD
empty = not great

preferred option is *make sure the variable is assigned before the template layer* but ~given the number of places assign('tabHeader', $someVar) happens that looks non trivial~ and we have a function for that! `addExpectedTabHeaderKeys` here we come :)